rotkehlchen/externalapis/etherscan.py

```diff
@@ -355,7 +355,7 @@ def get_transactions(
                 options['txHash'] = period_or_hash.hex()
                 parent_tx_hash = period_or_hash
 
-        transactions: Union[Sequence[EvmTransaction], Sequence[EvmInternalTransaction]] = []  # noqa: E501
+        transactions: Union[list[EvmTransaction], list[EvmInternalTransaction]] = []  # type: ignore  # noqa: E501
```

Comment:
why this change?

Reply:
because this was the least type ignores to make it work. list[None] != list[X] in mypy